### PR TITLE
WIP: About dialog Add system info

### LINKED ISSUE #5419: About dialog Add system info
**Do you want to request a *feature* or report a *bug*?**
Feature
**What is the current behavior?**
Have to run all over the place to get the current info for the environment section.
**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**

**What is the expected behavior?**
Add a feature that generates the following with filled in info in the about dialog.
```
- GitExtensions version:
- GIT version: 
- OS version: 
- .NET version: 
```
**Environment you encounter the issue:**

- GitExtensions version:
- GIT version: 
- OS version: 
- .NET version: 

**Did this work in previous version of GitExtensions (which)?**
No.

### PR DESCRIPTION
**Pull request by Aceto1**
Fixes #5419
 
Screenshots before and after (if PR changes UI):
Before:
image

After:
image
copied values:
image

What did I do to test the code and ensure quality:
- Fixed errors marked by linter

Has been tested on (remove any that don't apply):
- GIT 2.18.0.1
- Windows 10

TODO: 
Copy infos to clipboard when creating an issue

[EDIT] Added images directly

### CODE REVIEW

**Comment by vbjay** [comment]
You can paste screenshots straight into github pull requests and issues.  No need for links.

**Comment by vbjay** [comment]
FYI.  It is better to create a branch that contains your work so that if your master changes, the pull request doesn't change.  

It's
```
- GitExtension>>>>>s<<<<<<< version:
- GIT version: 
- OS version: 
- .NET version: 

```

File: GitUI/CommandsDialogs/FormAbout.Designer.cs [hunk 1]
```diff
@@ -229,6 +217,64 @@ private void InitializeComponent()
...
+            this.gitVersion.Text = "- GIT version: {0}";
+            // 
+            // gitExtensionsVersion
+            // 
+            this.gitExtensionsVersion.AutoSize = true;
+            this.gitExtensionsVersion.Location = new System.Drawing.Point(15, 76);
+            this.gitExtensionsVersion.Name = "gitExtensionsVersion";
+            this.gitExtensionsVersion.Size = new System.Drawing.Size(141, 15);
+            this.gitExtensionsVersion.TabIndex = 8;
+            this.gitExtensionsVersion.Text = "- GitExtension version: {0}";
```
Comment (vbjay) [inline GitUI/CommandsDialogs/FormAbout.Designer.cs]: It's "GitExtensions version:"

File: GitUI/CommandsDialogs/FormAbout.cs [hunk 2]
```diff
@@ -19,13 +22,19 @@ public FormAbout()
             InitializeComponent();
             InitializeComplete();
 
-            _NO_TRANSLATE_labelVersionInfo.Text += AppSettings.ProductVersion;
+            gitExtensionsVersion.Text = string.Format(gitExtensionsVersion.Text, AppSettings.ProductVersion);
```
Comment (vbjay) [inline GitUI/CommandsDialogs/FormAbout.cs]:
```c#
gitExtensionsVersion.Text = $"- GitExtensions version: {AppSettings.ProductVersion}";
```
and the same for the other values.

**Comment by gerhardol** [comment]
GE version should include the commit and maybe branch (helps for master builds). Too bad it is not available (I started that some time ago).

**Comment by RussKie** [comment]
@Aceto1 thank you for your contribution. 

As Jay said above, always create a new branch before starting work. Working on main/reserved branches (such as "master" or "release/xx") is never a good idea. If those change you will find yourself in a world of pain.

WRT to the submission, in principal I like the idea, but I'd like the UI tweaked a little bit. Something like this:

image

Please reduce the spacing between lines to be the same as in other paragraphs

**Comment by vbjay** [comment]
Any Update?

**Pull request closed without merging**